README.md

```diff
@@ -0,0 +1,126 @@
+# `dss-conduits`
+
+Part of this code was inspired by https://github.com/makerdao/rwa-toolkit/blob/master/src/urns/RwaUrn.sol mainly authored by livnev and https://github.com/dapphub/ds-roles/blob/master/src/roles.sol authored by DappHub.
+Since it should belong to the MakerDAO community the Copyright from our additions has been transferred to Dai Foundation.
+
+## Overview
+Implementation of the allocation system, based on the [technical specification forum post](https://forum.makerdao.com/t/preliminary-technical-specification-of-the-allocation-system/20921
+).
+
+![Untitled](https://github.com/makerdao/dss-allocator/assets/130549691/388f20fa-2d0c-484b-b716-fe4fa742115b)
+
+
+## Layers
+The system is comprised of several layers:
+
+- Core Allocation System (*green above*):
+    - Smart contracts that can be considered a part of the Maker Core Protocol, and are immutable and present in all Allocators.
+    - Their main role is to mint NST (New Stable Token) and hold it (possibly with other tokens) in the AllocationBuffer.
+- Deployment Funnels (*blue above*):
+    - Contracts that pull funds from the AllocatorBuffer.
+    - The funds can either be swapped or deployed into AMM pools or specific conduits.
+    - A typical setting for a funnel includes a base rate limited contract (such as Swapper) and an automation contract on top of it (such as StableSwapper).
+- Conduits (*orange above*):
+    - Yield investment singletons that support deposits and withdrawals.
+
+## Actors
+The allocation system includes several actor types:
+
+- Pause Proxy:
+    - Performs actions through spells with governance delay.
+    - In charge of setting up the core components and the NST minting instant access modules (DC-IAMs).
+    - Ward of the singleton contracts (e.g RWA conduits, Coinbase Custody, AllocatorRoles).
+- AllocatorDao Proxy:
```

Comment:
nit: this is sometimes spelled "AllocatorDao" and sometimes "AllocatorDAO" in this README, we should maybe use the latter form everywhere.

Reply:
Done.